### PR TITLE
Add 1 example

### PR DESCRIPTION
**Pull request by juancarlospaco**
- Add 1 example about ranges, tiny diff.

### CODE REVIEW

File: jekyll/code_carrousel.html [hunk 1]
```diff
@@ -220,5 +222,42 @@
...
 
+<code id="sample5" class="sample">
+{% highlight nim %}
+# int range, value must be between '5' and '8'.
+var variable0: 5..8 = 5
+variable0 = 8
+variable0 = 7
+assert not compiles(variable0 = 4)
+assert not compiles(variable0 = 9)
+# float range, value must be between '3.3' and '7.5'.
```
Comment (dom96) [inline jekyll/code_carrousel.html:233]:
```suggestion

# float range, value must be between '3.3' and '7.5'.
```

File: jekyll/code_carrousel.html [hunk 2]
```diff
@@ -220,5 +222,42 @@
...
+var variable1: 3.3..7.5 = 3.3
+variable1 = 7.5
+variable1 = 5.5
+assert not compiles(variable1 = 3.2)
+assert not compiles(variable1 = 7.6)
+# char range, value must be between 'b' and 'f'.
```
Comment (dom96) [inline jekyll/code_carrousel.html:239]:
```suggestion

# char range, value must be between 'b' and 'f'.
```

File: jekyll/code_carrousel.html [hunk 3]
```diff
@@ -220,5 +222,42 @@
...
+var variable2: 'b'..'f' = 'b'
+variable2 = 'f'
+variable2 = 'c'
+assert not compiles(variable2 = 'g')
+assert not compiles(variable2 = 'a')
+# Positive, value must be > 0.
```
Comment (dom96) [inline jekyll/code_carrousel.html:245]:
```suggestion

# Positive, value must be > 0.
```

File: jekyll/code_carrousel.html [hunk 4]
```diff
@@ -220,5 +222,42 @@
...
+var variable3: Positive = 1
+variable3 = 1
+variable3 = 999
+assert not compiles(variable3 = 0)
+assert not compiles(variable2 = -1)
+assert not compiles(variable2 = -9)
+# Natural, value must be >= 0.
```
Comment (dom96) [inline jekyll/code_carrousel.html:252]:
```suggestion

# Natural, value must be >= 0.
```

**Review by dom96 — CHANGES_REQUESTED** [review]
Dunno if we should add it, it just looks too much like a test suite for ranges. Maybe a short and sweet example would be better?

**Comment by juancarlospaco** [comment]
It is 2 `assert` each range, is not too much, and people are asking for more examples. 😐

**Comment by narimiran** [comment]
> it just looks too much like a test suite for ranges. Maybe a short and sweet example would be better?

I agree, it is too long/boring. Just keep int range, char range and `Natural`.

**Pull request closed without merging**